apps/scratch/src/hooks/useEmbeddedScratch.ts

```diff
@@ -28,16 +35,53 @@ const logModule = "[Embedded Scratch]";
 const messages = defineMessages({
   cannotLoadProject: {
     id: "useEmbeddedScratch.cannotLoadProject",
-    defaultMessage: "Could not load the project",
+    defaultMessage: "Could not load the project.",
+  },
+  projectJsonMissing: {
+    id: "useEmbeddedScratch.projectJsonMissing",
+    defaultMessage: "The project file is missing project.json.",
+  },
+  missingAssets: {
+    id: "useEmbeddedScratch.missingAssets",
+    defaultMessage: "The project is missing the following assets: {assets}.",
+  },
+  invalidScratchProject: {
+    id: "useEmbeddedScratch.invalidScratchProject",
+    defaultMessage: "The project file is not a valid Scratch project.",
```

Comment:
Unless I am missing something, the last comment still holds and these strings do not show up in the translation files, right?